src/Tool/ConfigDumperCommand.php

```diff
@@ -21,6 +20,17 @@
 use const STDERR;
 use const STDOUT;
 
+/**
+ * @psalm-type HelpObject = object{command:string}
+ * @psalm-type ErrorObject = object{command:string, message:string}
+ * @psalm-type ArgumentObject = object{
+ *     command:string,
+ *     configFile:string,
+ *     config:array<array-key, mixed>,
+ *     class:string,
```

Comment:
I was super confused by the lack of spacing here - was reading this as `class-string` @_@